scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/TimedDoTaskAction.java

```diff
@@ -86,14 +86,17 @@
 
     private final InternalTaskParentFinder internalTaskParentFinder;
 
+    private Map<TaskId, TaskResult> taskResults;
+
     /**
      * Create a new instance of TimedDoTaskAction
      *
      * @param launcher the launcher of the task
+     * @param taskResults parent tasks results
      */
     public TimedDoTaskAction(InternalJob job, TaskDescriptor taskDescriptor, TaskLauncher launcher,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)